lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -110,6 +116,33 @@ def oauth_cannot_be_secure(manifest)
           end
         end
 
+        def invalid_secure_param_scopes_error(manifest)
```

Comment:
This method seems to return many errors, how about renaming it to `invalid_secure_param_scopes_errors` in plural?